packages/cpp/src/browser/cpp-client-contribution.ts

```diff
@@ -27,4 +33,21 @@ export class CppClientContribution extends BaseLanguageClientContribution {
         ];
     }
 
+    protected createOptions(): LanguageClientOptions {
+        const clientOptions = super.createOptions();
+        clientOptions.initializationFailedHandler = () => {
+            const READ_INSTRUCTIONS_ACTION = "Read Instructions";
+            const ERROR_MESSAGE = "Error starting C/C++ language server. " +
```

Comment:
This will all be in a single line..? Is there enought space to see it ?

Reply:
Yes, it gets wrapped by the div or console.